modules/ROOT/pages/release-notes.adoc

```diff
@@ -23,6 +23,10 @@ These credentials can be consulted and modified:
 * in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
 * using xref:bonita-docker-installation.adoc#_monitoring_username[specific Environment parameters] when starting a *Bonita Docker image*
 
+=== Other changes
+
+* When the engine fails to evaluate output operations of a connector, the connector is no more automatically retried. It is put as failed to avoid automatic re-execution.
```

Comment:
We should explain that the decision is for the Customer to make, to re-execute it or not.
And precise the impacts of re-executing it.